concrete/src/Http/Middleware/ThumbnailMiddleware.php

```diff
@@ -108,24 +109,27 @@ private function generateThumbnails($paths, Connection $database)
     }
 
     /**
-     * Try building an unbuild thumbnail
+     * Try building an unbuilt thumbnail.
+     *
      * @param \Concrete\Core\Entity\File\File $file
-     * @param array $thumbnail
+     * @param array                           $thumbnail
+     *
      * @return bool
      */
     private function attemptBuild(File $file, array $thumbnail)
     {
         // If the file is already built, return early
         if ($this->isBuilt($file, $thumbnail)) {
             $this->completeBuild($file, $thumbnail);
+
             return true;
         }
 
         // Otherwise lets attempt to build it
         if ($dimensions = $this->getDimensions($thumbnail)) {
             list($width, $height, $crop) = $dimensions;
-
-            $this->getThumbnailer()->getThumbnail($file, $width, $height, (bool)$crop);
+            $type = new CustomThumbnail($width, $height, $thumbnail['path'], $crop);
```

Comment:
Im truly not sure, I don't see off the top of my head why you would override this,